packages/main/src/features/InputSuggestions.ts

```diff
@@ -81,15 +81,17 @@ class Suggestions {
 		this.selectedItemIndex = -1;
 	}
 
-	onUp(e: KeyboardEvent) {
+	onUp(e: KeyboardEvent, indexOfItem: number) {
 		e.preventDefault();
-		this._handleItemNavigation(false /* forward */);
+		indexOfItem = !this.isOpened && this._hasValueState && indexOfItem === -1 ? 0 : indexOfItem;
+		this._handleItemNavigation(false /* forward */, indexOfItem);
 		return true;
 	}
 
-	onDown(e: KeyboardEvent) {
+	onDown(e: KeyboardEvent, indexOfItem: number) {
 		e.preventDefault();
-		this._handleItemNavigation(true /* forward */);
+		indexOfItem = !this.isOpened && this._hasValueState && indexOfItem === -1 ? 0 : indexOfItem;
```

Comment:
I think it is better approach to not modify the input parameter "indexOfItem" but rather create a new constant and pass it to this._handleItemNavigation()